### ISSUE TITLE
Add support for Arch Linux

### ISSUE THREAD

**Issue by mubaris** [tliron/install-gnome-themes#5]
I am running on Apricity OS - Gnome 3.24

Here's the output.

```
Detected GTK .
This script only GTK version 3.18 up to 3.24.
```

**Comment by tliron** [comment]
I would love to add support but do not normally use Arch. :/ Could anybody please contribute a fix for Arch?

**Comment by KrumpetPirate** [comment]
utils.sh would have to have the following for gtk-version to be correctly captured:
`VERSION=$(pacman -Q gtk3 | cut --delimiter=' ' --fields=2 | cut --delimiter='.' --fields=1,2)`

Since the script only supports gtk3, I'm not sure why there is a check for gtk2 being installed...

I am not sure if there is a way to check gtk version without using dpkg or the package manager, anyone else know?

**Comment by tliron** [comment]
The GTK2 is just a fallback so that something useful at least would be presented to the user.

I don't think it's possible to check it without a package manager. After all, you might have both GTK2 and GTK3 installed at the same time, so there's no one version used. GTK1 had a `gtk-config` tool, but it was depracated and removed.

Anyway, GTK version is not enough for Arch support -- we also need to install all the required dependencies for building and using the themes. If you can send a PR that would be great, but even if you just hack the file and get it to work and send it to me I can merge it.

**Comment by KrumpetPirate** [comment]
Yes you are right. I will have to look at it later when I have some time.

**Comment by KrumpetPirate** [comment]
I'm afraid I'm going to bork my install if I continue on this. Arch is so wide open as well that it makes it a little harder. Manjaro might have some pre-built packages that Antergros doesn't have and for sure Arch doesn't have. Someone better than I will have to give this a go.

**Comment by gnacho** [comment]
Same issue with ApricityOS :(

**Comment by Findarato** [comment]
I have this running on fedora by using the following command. 

```bash
rpm -ql --info gtk3-devel 2>/dev/null | grep '^Version' | cut --delimiter=' ' --fields=7- | cut --delimiter='.' --fields=1,2
```

I might try and get this working on Manjaro and Antergros if I have time today

**Comment by tliron** [comment]
I have removed all OS-specific code from the script as it was getting unmanageable. Instructions are now provided in the README.

If you'd like to contribute instructions for Arch, they would be much appreciated!

**Comment by tliron** [comment]
Anybody want to contribute a `install-requirements-arch` file?
